tests/integration_tests/test_examples.py

```diff
@@ -54,6 +54,50 @@ def test_algo_examples(filepath):
     assert subprocess.run([filepath], check=False, env=env).returncode == 0
 
 
+@pytest.mark.no_cover
+@pytest.mark.timeout(180)
+def test_dqn_pong():
+    """Test tf/dqn_pong.py with reduced replay buffer size for reduced memory
+    consumption.
+    """
+    env = os.environ.copy()
+    env['GARAGE_EXAMPLE_TEST_N_EPOCHS'] = '1'
+    assert subprocess.run(
+        [str(EXAMPLES_ROOT_DIR / 'tf/dqn_pong.py'), '--buffer_size', '5'],
+        check=False,
+        env=env).returncode == 0
+
+
+@pytest.mark.no_cover
+@pytest.mark.timeout(20)
+def test_ppo_memorize_digits():
+    """Test tf/ppo_memorize_digits.py with reduced batch size for reduced memory
+    consumption.
+    """
+    env = os.environ.copy()
+    env['GARAGE_EXAMPLE_TEST_N_EPOCHS'] = '1'
+    assert subprocess.run([
+        str(EXAMPLES_ROOT_DIR / 'tf/ppo_memorize_digits.py'), '--batch_size',
+        '4'
+    ],
+                          check=False,
```

Comment:
weird spacing

Reply:
yapf wasn't having it, so I extracted popenargs into a variable